### PR TITLE
build ocp repo names dynamically

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar <shmohan@redhat.com>

### CODE REVIEW

**Comment by shylesh** [comment]
Rebased on latest master

**Comment by shylesh** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/5044/ : Deployment has been successful in this run

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -1061,6 +1061,20 @@ def get_ocp_version(seperator=None):
     )
 
 
+def get_ocp_repo():
+    """
+    Get ocp repo file, name will be generated dynamically based on
+    ocp version.
+
+    Returns:
+        string : Path to ocp repo file
```
Comment (vavuthu) [inline ocs_ci/utility/utils.py]: leave blank line after line 1070

**Comment by shylesh** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/5084/
Deployment test passed.

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -1061,6 +1061,21 @@ def get_ocp_version(seperator=None):
...
+
+    """
+    repo_path = os.path.join(
+        constants.REPO_DIR, f"ocp_{get_ocp_version('_')}.repo"
```
Comment (clacroix12) [inline ocs_ci/utility/utils.py]: As we only have `ocp_4_2.repo` and `ocp_4_3.repo` currently, doesn't this reintroduce the issue where versions like `4.2.0` or `4.3.1` will fail due to not having repo files?

Comment (shylesh) [reply ocs_ci/utility/utils.py]: I am not sure whether we need seperate repo files for minor-minor versions so in both aws and vmware we were assuming for any 4.2.x the repo file will be ocp_4_2.repo.

If we need seperate repo file for each of  4.2.x etc, Vijay already had code to get minor-build versions and by having corresponding repo file would work for us.

@vavuthu @petr-balogh Any inputs on seperate repo file requirement for each minor-minor versions.

Comment (vavuthu) [reply ocs_ci/utility/utils.py]: I had same question when I worked initially, The response from ocp team is when there is a new build for errata, this repo will be updated. So there is no need for extra repo for batch versions

**Pull request merged**